staging/src/k8s.io/apiserver/pkg/server/config.go

```diff
@@ -260,9 +263,15 @@ func (c *Config) ApplySecureServingOptions(secureServing *options.SecureServingO
 		return c, nil
 	}
 
+	ipMax := c.MaxRequestsInFlight / 2
+	if ipMax > c.MaxMutatingRequestsInFlight/2 {
+		ipMax = c.MaxMutatingRequestsInFlight / 2
+	}
+	max := c.MaxRequestsInFlight + c.MaxMutatingRequestsInFlight
```

Comment:
Agree we shouldn't have any completely unbounded counts, I just don't have a good sense of the number of long running connections we have today (for example, in the 5k kubemark tests), so I'm not sure what to set the initial limit at